### PR TITLE
[Issue-400] Implementing dynamic alpha for coefplot and iplot

### PR DESCRIPTION
**Pull request by rafimikail**

### CODE REVIEW

**Comment by rafimikail** [comment]
Hi @s3alfisc , this will be the merge request for implementing dynamic alpha for iplot and coefplot

I will re-do a sanity check, remove unnecessary files (used for testing), and put the proof of the test later

Thanks

File: pyfixest/estimation/feols_.py [hunk 1]
```diff
@@ -1503,8 +1503,14 @@ def tidy(self, alpha=0.05) -> pd.DataFrame:
             A tidy pd.DataFrame containing the regression results, including point
             estimates, standard errors, t-statistics, and p-values.
         """
-        _coefnames = self._coefnames
+        if alpha is None:
+            ub, lb = 0.975, 0.025
+            self.get_inference()
```
Comment (rafimikail) [inline pyfixest/estimation/feols_.py:1508]: I recall the get_inference as if the user had inputted non-default alpha for tidy previously, then the user recall tidy again, the value of confidence interval will be the non-default eventho the dataframe columns will show a default confidence interval

What do you think? @s3alfisc

**Comment by rafimikail** [comment]
Hi @s3alfisc , kindly need your review for this merge request, I've also included the test results. Let me know if there are any improvements or changes needed. Thanks!

Test proofs:
1. Coefplot using default alpha value
2. Coefplot using adjusted alpha value
3. Iplot using adjusted alpha value


image

image

image

File: pyfixest/estimation/feols_.py [hunk 2]
```diff
@@ -1503,8 +1503,14 @@ def tidy(self, alpha=0.05) -> pd.DataFrame:
...
+        else:
+            ub, lb = 1 - alpha / 2, alpha / 2
+            self.get_inference(alpha=1 - alpha)
```
[the remark this answers is not in the document]
Comment (s3alfisc) [reply pyfixest/estimation/feols_.py:1518]: Solved by [695ad80](https://github.com/py-econometrics/pyfixest/pull/528/commits/695ad8093d3c20d1b3bd10faf086170cd3a39456)[695ad80](https://github.com/py-econometrics/pyfixest/pull/528/commits/695ad8093d3c20d1b3bd10faf086170cd3a39456)

File: pyfixest/estimation/feols_.py [hunk 3]
```diff
@@ -1490,7 +1490,7 @@ def get_performance(self) -> None:
         self._adj_r2 = np.nan
         self._adj_r2_within = np.nan
 
-    def tidy(self, alpha=0.05) -> pd.DataFrame:
+    def tidy(self, alpha: Optional[float] = None) -> pd.DataFrame:
         """
         Tidy model outputs.
```
[the remark this answers is not in the document]
Comment (s3alfisc) [reply pyfixest/estimation/feols_.py:1497]: Solved by [695ad80](https://github.com/py-econometrics/pyfixest/pull/528/commits/695ad8093d3c20d1b3bd10faf086170cd3a39456)[695ad80](https://github.com/py-econometrics/pyfixest/pull/528/commits/695ad8093d3c20d1b3bd10faf086170cd3a39456)

**Comment by s3alfisc** [comment]
Hi @rafimikail , I made minor adjustments. Will merge this after the CI tests pass =) Congrats 🎉 and thanks for your first contribution to pyfixest!

**Comment by s3alfisc** [comment]
@all-contributors please add @rafimikail for code

**Comment by rafimikail** [comment]
Hi @s3alfisc , I appreciate your help with the alpha consistency in get_inference, I was actually aware of it but I was concerned it might have dependencies with other functions, so i did not change it directly. Sorry could not address it myself immediately, it was quite late here in my country when you commented.

Also Big Thanks! I'm really pleased with my first contribution and I am interested to keep contributing!

**Pull request merged**